environmentmodel/environment-model_shacl.ttl

```diff
@@ -0,0 +1,288 @@
+@prefix environmentmodel: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/environmentmodel/> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+environmentmodel:environmentModelShape a sh:NodeShape ;
+    sh:property [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node environmentmodel:QuantityShape ;
+            sh:order 2 ;
+            sh:path environmentmodel:Quantity ],
```

Comment:
attributes should be camelCase
-> `sh:path environmentmodel:quantity`

Reply:
I adapted that.